sdk/dataprotection/Azure.ResourceManager.DataProtectionBackup/src/Generated/ResourceGuardResource.cs

```diff
@@ -472,7 +472,7 @@ public virtual Pageable<ResourceGuardProtectedObjectData> GetDeleteResourceGuard
         /// </item>
         /// <item>
         /// <term>Operation Id</term>
-        /// <description>ResourceGuards_GetBackupSecurityPinRequestsObjects</description>
+        /// <description>ResourceGuards_GetBackupSecurityPINRequestsObjects</description>
```

Comment:
it's `PIN`: https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/dataprotection/resource-manager/Microsoft.DataProtection/stable/2024-04-01/dataprotection.json#L3395